src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+// configureStore.js
+
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './Books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
```

Comment:
- [ ] CategoriesReducer should be added here as already pointed out above :point_up: